_layouts/default.html

```
@@ -13,14 +13,19 @@
 
     {{ content }}
 
-    <footer>
+    <footer class='buffer-footer'>
       <div class="row">
         <div class="large-12 columns">
-          <a href="https://github.com/emacs-berlin/emacs-berlin.org" target="_blank">Site source code is available on Github.</a>
+          <a href="https://github.com/emacs-berlin/emacs-berlin.org" target="_blank">Site source code is available on GitHub</a>
         </div>
       </div>
     </footer>
 
+    <footer class='mode-line'>
+      U:--- {{page.filename}}&lt;emacs-berlin.org&gt;&nbsp;&nbsp;&nbsp;Git:gh-pages&nbsp;&nbsp;(Markdown Jekyll)
```

this will probably fail hard when it is not not set in a file.